src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -839,9 +827,9 @@ int mongoSubCacheItemInsert(const char* tenant, const BSONObj& sub, const char*
   std::vector<BSONElement>  attrVec       = sub.getField(CSUB_ATTRS).Array();
   std::vector<BSONElement>  condVec       = sub.getField(CSUB_CONDITIONS).Array();
 
-  if ((lastNotificationTime == -1) && (sub.hasField(CSUB_LASTNOTIFICATION)))
+  if (lastNotificationTime == -1)
```

Comment:
Could you elaborate on this part, please? Maybe we have discussed at skype, but I cannot remember about this no...